doc/nrf/ug_matter_configuring_protocol.rst

```diff
@@ -53,23 +43,41 @@ This option sets the :kconfig:`CONFIG_OPENTHREAD_MTD_SED` Kconfig option.
 
 For more information about configuring OpenThread in the |NCS|, see :ref:`ug_thread_configuring`.
 
+.. _ug_matter_configuring_optional_nfc:
+
 Commissioning with NFC
 ======================
 
 You can configure the Matter protocol to use NFC tag for :doc:`commissioning <matter:nrfconnect_android_commissioning>`, instead of the default QR code.
 
 To enable NFC for commissioning and share the onboarding payload in an NFC tag, set the :kconfig:`CONFIG_CHIP_NFC_COMMISSIONING` Kconfig option.
 
+.. _ug_matter_configuring_optional_log:
+
 Logging configuration
 =====================
 
 Logging is handled with the :kconfig:`CONFIG_LOG` option.
 This option enables logging for both the stack and Zephyr's :ref:`zephyr:logging_api` API.
 
+.. _ug_matter_configuring_optional_shell:
+
 Matter shell commands
 =====================
 
 You can enable the Matter shell library using the :kconfig:`CONFIG_CHIP_LIB_SHELL` Kconfig option.
 This option lets you use the Matter shell commands with :ref:`matter_samples`.
 
 See :doc:`matter:nrfconnect_examples_cli` in the Matter documentation for the list of available Matter shell commands.
+
+.. _ug_matter_configuring_requirements:
+
+Required components for Matter network
```

Comment:
Is this section really required? The page is about configuring a Matter application which is unrelated to Matter controllers.

Reply:
This is meant as "what I do next" section. Maybe we need to name it differently.

Reply:
Yeah, but if someone reads about changing the Matter app configuration he's probably already done with the environment setup, so I wonder if this information is useful.

Reply:
But it's just my opinion. Feel free to disregard :)

Reply:
No changes made.

Reply:
Ok, I'll remember that.